packages/css/src/plugins/atomicify-rules.tsx

```diff
@@ -102,19 +101,24 @@ const buildAtomicSelector = (node: Declaration, opts: AtomicifyOpts) => {
  * @param node
  * @param opts
  */
-const atomicifyDecl = (node: Declaration, opts: AtomicifyOpts) => {
+const atomicifyDecl = (node: Declaration, opts: AtomicifyOpts, postcss: Postcss) => {
   const selector = buildAtomicSelector(node, opts);
-  const newDecl = decl({ prop: node.prop, value: node.value });
-  const newRule = rule({ selector, nodes: [newDecl] });
-
-  // Pass on important flag.
-  newDecl.important = node.important;
+  const newDecl = postcss.decl({
+    raws: { before: '', value: { value: '', raw: '' }, between: '' },
+    // Pass on important flag.
+    important: node.important,
+    prop: node.prop,
+    value: node.value,
+  });
+  const newRule = postcss.rule({
+    raws: { before: '', after: '', between: '', selector: { raw: '', value: '' } },
+    nodes: [newDecl],
+    selector,
+  });
 
   // We need to link the new node to a parent else autoprefixer blows up.
   newDecl.parent = newRule;
-  newDecl.raws.before = '';
   newRule.parent = opts.parentNode!;
```

Comment:
Just to be a bit cleaner, you should be able to place this inside `newRule` as a key. It seems to be overridden from https://github.com/postcss/postcss/blob/main/lib/node.js#L52 which `rule` extends from

Reply:
parent doesn't seem to exist on the RuleProps type, just the Rule type after creation